doc/Publishing.md

```diff
@@ -142,3 +142,10 @@ Updating the `yarn.lock` helps identify potential problems with our dependency v
 To successfully complete a `yarn upgrade`, one must:
 - perform `yarn upgrade` at the root of the repository.
 - fix any compilation errors, typing issues, and failing tests that may be introduced.
+
+### Update Milestones
+
+* Close current release milestone.
+* Create next two milestones, in the case they are not already exsting. Date is last Thursday of the month. Exceptions, e.g. on Christmas are to be considered.
```

Comment:
We can probably omit the following:

>  Date is last Thursday of the month.

and

> Exceptions, e.g. on Christmas are to be considered.

These two points are related to the entire release, not solely the milestones.

Reply:
They are, but I think it is useful to have this information right here where it is needed (to created the milestones). Is there any other place where this is documented?

Reply:
Likely not explicitly, the release is generally performed on the last Thursday of every month unless stated otherwise (dev meeting, discourse). That is why if we are to add that note it should probably be at the beginning of the procedure and not one of the final points.

Reply:
I argue that the point I add is actually the the first step of the next release at the same time as it is the last of the current one :-)
More seriously, I really believe that the information is useful at exactly this step where you create the future milestone and therefore set the date for the release.